sources/platform/workflow/job/runsh.md

```diff
@@ -32,7 +32,7 @@ jobs:
         versionName:    <name of the version you want to pin>
       - IN:         <resource>
         versionNumber:    <number of the version you want to pin>        
-      - IN:         <gitRepoResource with buildOnPullRequest: true>
+      - IN:         <gitRepoResource with buildOnPullRequest: true and buildOnPullRequestClose: true>
```

Comment:
i dont think we should add it here either, this is just an example of what can be used as IN to runSh, so we don't need to list all the options.